uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/manager-leave/components/manager-leave-calendar.js

```diff
@@ -506,15 +530,16 @@ define([
      */
     function setCalendarProps(contactID, calendar) {
       var leaveRequest,
-        indexedByMonth = vm.months.map(function () {
-          return [];
+        monthObject,
+        indexedByMonth = _.map(vm.months, function (month) {
+          return _.extend(month, {
+            data: []
```

Comment:
Month is used to only show the same in the calendar. But data cannot be attached to that, because data is needed for each contact. So I am making a copy of `month` and adding `data` which will be stored for each contact.